graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -186,19 +216,21 @@ class RenameSchemaTypesVisitor(Visitor):
         }
     )
 
-    def __init__(self, renamings, query_type, scalar_types):
+    def __init__(
+        self, renamings: Dict[str, Optional[str]], query_type: str, scalar_types: Set[str]
+    ):
         """Create a visitor for renaming types in a schema AST.
 
         Args:
-            renamings: Dict[str, str], mapping from original type name to renamed type name.
-                       Any name not in the dict will be unchanged
+            renamings: Dict[str, Optional[str]], mapping from original type name to renamed type
+            name. Any name not in the dict will be unchanged
```

Comment:
Similarly to the above, please make sure your code edits match and preserve the style of the surrounding code. Here, line 226 gets de-indented, even though the line it replaced and the next few lines have a different style.

It's often very useful to give your own code a self-review in the PR review viewer before having others review it — these things are much more obvious when looking at a diff, and I'm sure you'll be able to catch most if not all of them by yourself. That will speed up the review process drastically, and make you and the team more productive.